dashboard/app/assets/javascripts/videos.js.erb

```diff
@@ -135,7 +135,11 @@ function setupVideoFallback(videoInfo, playerWidth, playerHeight, shouldStillAdd
   }
 
   if (videoInfo['force_fallback']) {
-    addFallbackVideoPlayer(videoInfo, playerWidth, playerHeight);
+    // this and the below setTimeout wrapper work around a load-ordering issue, see detailed comment:
+    // https://github.com/code-dot-org/code-dot-org/pull/3260#discussion_r35707401
+    setTimeout(function() {
+      addFallbackVideoPlayer(videoInfo, playerWidth, playerHeight);
```

Comment:
I read the detailed comment but still don't fully understand the scenario.

Is it absolutely guaranteed that the iframe element will exist when the timeout function is called, or just very likely?  Is there some event that you can fire on that will guarantee that the needed iframe has been created?